### ISSUE TITLE
md-modal options not applied

### ISSUE THREAD

**Issue by avrahamcool** [aurelia-ui-toolkits/aurelia-materialize-bridge#393]
setting md-modal `starting-top` option is not working as expected
[This gist is runing against 0.25.0](https://gist.run/?id=670609185399bf0904f5905c73639a15)

in the gist, I show that 2 modals are acting in the same way, even if you set the `starting-top` to different values.

**Comment by Thanood** [comment]
Interesting, thanks for mentioning. Although I'm not sure what's wrong, the options are simply passed to the Materialize function.

**Comment by Thanood** [comment]
I can reproduce this but until now, I cannot see what's wrong..

The modal is initialized with the following options:

```json
"
{
  "dismissible":false,
  "endingTop":"20%",
  "inDuration":1000,
  "opacity":0.7,
  "outDuration":1000,
  "startingTop":"10%"
}
```

Which seems to work in "vanilla" Materialize..

**Comment by Thanood** [comment]
I didn't forget about this issue.
But still no idea what's going wrong.. 😕

**Comment by Thanood** [comment]
Maybe this is fixed with v0.100.0:

> Fixed bug where modal open did not use initialized options

I will try it.

**Comment by Thanood** [comment]
Fiexed in v0.100.1 ✨
